modules/microshift-lvm-thin-volumes.adoc

```diff
@@ -17,6 +17,11 @@ To use advanced storage features such as creating volume snapshots or volume clo
 To create Container Storage Interface (CSI) snapshots, you must configure thin volumes on the {op-system-ostree} host. The CSI does not support volume shrinking.
 ====
 
+[IMPORTANT]
+====
+When using thin provisioning, it is important that you monitor the storage pool and add more capacity as the available physical space runs out. You can configure the storage pool to auto expand when there is available space within the volume group (VG). See link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/configuring_and_managing_logical_volumes/index#creating-thinly-provisioned-logical-volumes_creating-and-managing-thin-provisioned-volumes[Creating thinly-provisioned logical volumes]
```

Comment:
```suggestion
When using thin provisioning, it is important that you monitor the storage pool and add more capacity as the available physical space runs out. You can configure the storage pool to auto expand when there is available space within the volume group (VG). See link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html-single/configuring_and_managing_logical_volumes/index#creating-thinly-provisioned-logical-volumes_creating-and-managing-thin-provisioned-volumes[Creating thinly provisioned logical volumes]
```
OK, RHEL might have this wrong, but no hyphen when there is an "ly." Let's make it correct here. I don't think that will confuse anyone.

Reply:
Done